tests/testthat/test_await.R

```diff
@@ -243,17 +243,18 @@ test_that("await_all calls f until completion - multivariate", {
 
 test_that("await_all returns list of completed responses - multivariate", {
   set.seed(2)
-  x <- await_all(f_rand, .x = 1:2, .y = 3:4, job_id = 1)
+  x <- await_all(f_rand, .x = 1:2, .y = 3:4)
   expect_is(x, "list")
   expect_equal(sapply(x, get_status), rep("succeeded", 2))
-  expect_equal(sapply(x, function(x) x$args), list(c(1, 3), c(2, 4)))
-  expect_equal(sapply(x, function(x) x$job_id), rep(1, 2))
+  expect_equal(lapply(x, function(x) attr(x, 'args')),
+               list(list(job_id = 1, run_id = 3),
+                    list(job_id = 2, run_id = 4)))
 })
 
 test_that("await_all catches arbitrary status and keys - multivariate", {
   f <- function(x, y) list(party_status = "going home", value = x)
   x <- await_all(f, .x = 1:2, .y = 3:4, .status_key = "party_status",
                  .success_states = "going home", .verbose = TRUE)
   expect_equal(sapply(x, get_status), rep("going home", 2))
-  expect_equal(sapply(x, function(x) x$value), cbind(c(1, 3), c(2, 4)))
```

Comment:
I'm not sure why this was ever the intended behavior, since `f` only returns `x` in the value slot.